CodeU_Assignement_5/src/codeu_assignement_5/KemUnknownLanguage.java

```diff
@@ -0,0 +1,145 @@
+package codeu_assignement_5;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author Karine
+ */
+public class KemUnknownLanguage {
+    /*
+     * Input: a dictionary (a list of words in lexicographic order) of all words in an unknown/invented language, 
+    *  Output: alphabet (an ordered list of characters) of that language, letters we cannot decide are appended in the end.
+    */
+    public List<Character> getAlphabeit(String[] alienWords) {
+        if ((alienWords != null) && (alienWords.length > 0)) {
+           return extractAlphabeitFromWords(alienWords);
+        }
+        
+        return null;
+    }
```

Comment:
Yes, I was trying to fix it several time. I will revert it to the original code. 
The other reviewer also comment on this section, why don't you comment together on the same place? else it is very hard (for me) to get an agreement between the two reviewers...